src/cordova/plugin/plugin.xml

```diff
@@ -51,16 +51,9 @@
 
     <hook type="before_plugin_install" src="android/scripts/copy_third_party.js" />
 
-    <source-file src="android/aidl" target-dir="app/src/main" />
     <source-file
       src="android/java/org/outline/OutlinePlugin.java"
       target-dir="app/src/main/java/org/outline" />
-    <source-file
-      src="android/java/org/outline/vpn"
-      target-dir="app/src/main/java/org/outline" />
-    <source-file
-      src="android/java/org/outline/log"
-      target-dir="app/src/main/java/org/outline" />
     <source-file
```

Comment:
Can we remove this? Isn't `OutlineAndroidLib` already pulling it?

Reply:
It looks like we can specify a local directory as a repository in Gradle, then refer to the framework by name.
See https://github.com/golang/go/wiki/Mobile#building-and-deploying-to-android-1:
![image](https://user-images.githubusercontent.com/113565/212126356-aeda26d9-5268-4ef5-bbae-670e26e28521.png)

I believe we can do that in `android/java/build-extras.gradle`

Reply:
We may be able to specify the dependency on build-extras only and remove the `framework` entry here.

Reply:
Oh, we need to add the AndroidLib to the list of repositories too.

Reply:
Unfortunately, `flatDir` is not compatible with composite builds, seemingly due to a bug in Gradle: https://github.com/gradle/gradle/issues/911.